clickhouse/tests/test_clickhouse.py

```diff
@@ -0,0 +1,63 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from datadog_checks.clickhouse import ClickhouseCheck
+
+from .common import CLICKHOUSE_VERSION
+from .metrics import ALL_METRICS
+
+pytestmark = [pytest.mark.integration, pytest.mark.usefixtures('dd_environment')]
+
+
+def test_check(aggregator, instance):
+    check = ClickhouseCheck('clickhouse', {}, [instance])
+    check.run()
+
+    server_tag = 'server:{}'.format(instance['server'])
+    port_tag = 'port:{}'.format(instance['port'])
+    for metric in ALL_METRICS:
+        aggregator.assert_metric_has_tag(metric, server_tag)
+        aggregator.assert_metric_has_tag(metric, port_tag)
+        aggregator.assert_metric_has_tag(metric, 'db:default')
+        aggregator.assert_metric_has_tag(metric, 'foo:bar')
+
+    aggregator.assert_metric('clickhouse.table.replicated.total', 2)
+    aggregator.assert_metric(
+        'clickhouse.dictionary.item.current', tags=[server_tag, port_tag, 'db:default', 'foo:bar', 'dictionary:test']
+    )
+
+
+def test_custom_queries(aggregator, instance):
```

Comment:
It'd be nice to add a test for custom queries errors. This is a part where users may iterate on, and good error feedback is important.